src/FontCache.cc

```diff
@@ -408,36 +410,50 @@ FT_Face FontCache::find_face_fontconfig(const std::string& font) const
   }
 
   FcValue font_index;
-  if (FcPatternGet(match, FC_INDEX, 0, &font_index)) {
+  if (FcPatternGet(match, FC_INDEX, 0, &font_index) != FcResultMatch) {
     return nullptr;
   }
 
-  FT_Face face;
-  const FT_Error error = FT_New_Face(this->library, (const char *) file_value.u.s, font_index.u.i, &face);
+  FcValue font_features;
+  std::string font_features_str;
+  if (FcPatternGet(match, FC_FONT_FEATURES, 0, &font_features) == FcResultMatch) {
```

Comment:
Maybe FcPatternGetString().

Reply:
The benefit seems superficial, but why not. They took the time to provide those :grinning:.

Reply:
Type safety is good.